modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Validation/Validation.js

```diff
@@ -120,7 +120,9 @@ const Validation = ({
 					name={name}
 					onBlur={onBlur}
 					parameter={parameter}
-					parentFieldName={validation.fieldName}
+					parentFieldName={
+						validation?.fieldName ?? focusedField?.fieldName
+					}
```

Comment:
@miltonmc Not the solution we desired, but we're blocked by the inconsistent behavior of the focusedField state

Reply:
It's ok as a quick fix, but we should sign that this solution should not be replicated.

Please extract this code  and the similar one in the reducer to a variable and add a `TODO` explaining why we did that and what we should expect instead:
```javascript
/* TODO: Remove this variable when the fieldName is properly delivered by the validator prop */
const parentFieldName = validation?.fieldName ?? focusedField.fieldName;
```